housing_cooperative_base/models/lease.py

```diff
@@ -197,3 +225,17 @@ def _default_journal(self):
         )
         domain = [("type", "=", "sale"), ("company_id", "=", company_id)]
         return self.env["account.journal"].search(domain, limit=1)
```

Comment:
It'd be safer to check wether you found a journal and raise a `ValidationError` (?) if you don't.